core/spec/mailers/order_mailer_spec.rb

```diff
@@ -9,12 +9,12 @@
 
   let(:order) do
     order = stub_model(Spree::Order)
-    product = stub_model(Spree::Product, :name => %Q{The "BEST" product})
-    variant = stub_model(Spree::Variant, :product => product)
-    price = stub_model(Spree::Price, :variant => variant, :amount => 5.00)
-    line_item = stub_model(Spree::LineItem, :variant => variant, :order => order, :quantity => 1, :price => 4.99)
-    allow(variant).to receive_messages(:default_price => price)
-    allow(order).to receive_messages(:line_items => [line_item])
+    product = stub_model(Spree::Product, name: %Q{The "BEST" product})
```

Comment:
Use `%` instead of `%Q`.<br>Do not use `%Q` unless interpolation is needed.  Use `%q`.<br>Use `%Q` only for strings that contain both single quotes and double quotes, or for dynamic strings that contain double quotes.